### PR TITLE
feat(react): add sentry and sentry middleware example

### PR DESCRIPTION
**Pull request by jon-nona**
## Scope

PT5 - Sentry - Redux Sentry Middleware

## Work Done

- Add Sentry and Redux Sentry Middleware to web-ui-react

## Steps to Test

- pull the branch
- create an environment file and add a Sentry DSN environment file.
- Cause an application error and check that it captures the redux application state in Sentry

### CODE REVIEW

File: docs/branch/iso__react__redux__middleware__sentry__redux-sentry-middleware.tpl.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# React Redux Sentry
+
+> Implements Sentry and Redux Sentry middleware for React
```
Comment (neilrussell6) [inline docs/branch/iso__react__redux__middleware__sentry__redux-sentry-middleware.tpl.md:3]: please add reference to the module used with a link eg.
```
Implements Sentry and Redux Sentry middleware for React using [redux-sentry-middleware](https://github.com/vidit-sh/redux-sentry-middleware)
```

File: docs/branch/iso__react__redux__middleware__sentry__redux-sentry-middleware.tpl.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    +-- common
+        +-- sentry
+```
+
+[src/common/sentry/index.js](https://raw.githubusercontent.com/Nona-Creative/nona-playground/iso__react__redux__middleware__sentry__redux-sentry-middleware/packages/web-ui-react/src/common/sentry/index.js)
+
+{lang=javascript}
+<<[](../packages/web-ui-react/src/common/sentry/index.js)
+
+[src/common/sentry/index.js](https://raw.githubusercontent.com/Nona-Creative/nona-playground/iso__react__redux__middleware__sentry__redux-sentry-middleware/packages/web-ui-react/src/common/redux/middleware.js)
```
Comment (neilrussell6) [inline docs/branch/iso__react__redux__middleware__sentry__redux-sentry-middleware.tpl.md:28]: please update this to:
```
[src/common/redux/middleware.js](...
```

File: packages/web-ui-react/.env.tpl [hunk 3]
```diff
@@ -1,2 +1,5 @@
 # npm run help
 PACKAGE_JSON_HELP_PADDING=35
+
+# Sentry
+REACT_APP_SENTRY_DSN=xxxxxxxx
```
Comment (neilrussell6) [inline packages/web-ui-react/.env.tpl:5]: please change to this:
```
REACT_APP_SENTRY_DSN=
```
this is the convention I am using everywhere else on the playground and I think is more common

File: packages/web-ui-react/src/index.js [hunk 4]
```diff
@@ -1,7 +1,6 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 import { Provider } from 'react-redux'
-
```
Comment (neilrussell6) [inline packages/web-ui-react/src/index.js:4]: please re-add this newline

**Review by neilrussell6 — CHANGES_REQUESTED** [review]

File: docs/branch/iso__react__redux__middleware__sentry__redux-sentry-middleware.tpl.md (hunk 1 above)
Comment (jon-nona) [reply docs/branch/iso__react__redux__middleware__sentry__redux-sentry-middleware.tpl.md]: @neilrussell6 this should be addressed now

File: docs/branch/iso__react__redux__middleware__sentry__redux-sentry-middleware.tpl.md (hunk 2 above)
Comment (jon-nona) [reply docs/branch/iso__react__redux__middleware__sentry__redux-sentry-middleware.tpl.md]: done:)

**Review by neilrussell6 — APPROVED** [review]

**Pull request merged**